### ISSUE TITLE
Join recipe view with getRecipe method of the API

### ISSUE THREAD

**Issue by josecriane** [josecriane/cookingcheats#21]

**Comment by josecriane** [comment]
I think on two possible solutions.
1. the get request with php and return all recipe information on hidden divs.
2. the recipe page only return html, js send a recipe request using ajax. This option has a advantage, that recipe page could be seen as a template, and this page could be cached.
